src/Sylius/Bundle/ShopBundle/Twig/Component/Checkout/Address/FormComponent.php

```diff
@@ -86,6 +96,10 @@ public function addressFieldUpdated(#[LiveArg] mixed $addressId, #[LiveArg] stri
         $newAddress['city'] = $address->getCity();
         $newAddress['postcode'] = $address->getPostcode();
```

Comment:
I think we can make this logic conditional, move it into a private method, and then register the `DefaultAddressFormValuesModifier` with a high priority.

```suggestion
        if ($this->addressFormValuesModifiers === null) {
            $newAddress['firstName'] = $address->getFirstName();
            ...
        }
```

or

```suggestion
        if ($this->addressFormValuesModifiers === null) {
            $newAddress = $this->buildBaseAddress($address);
        }
```